common/defs.go

```diff
@@ -99,6 +108,9 @@ type PortUpdateMessage struct {
 	Port uint64 `json:"port"`
 }
 
+// Endpoint represents an endpoint - that is, something that
```

Comment:
I like this docstring. That's the sort of useful, descriptive docstring I'd like to see more of.